src/TraceParserGen/TraceParserGen.cs

```diff
@@ -314,6 +328,12 @@ private void GenerateTemplateDefs(TextWriter output)
         output.WriteLine("                    callback(template);");
         output.WriteLine("        }");
         output.WriteLine();
+
+        output.WriteLine("        private void RegisterTemplate(TraceEvent template)");
```

Comment:
I guess this is a compromise between the previous style and the current style. Personally, I like the current style, which omits this private method call but it isn't any different in practice